e2e-tests/wait-for-start.sh

```diff
@@ -1,7 +1,37 @@
 #!/usr/bin/env bash
 
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+######## WAIT FOR START #############
+# Script used in e2e-test that waits 
+# for OpenMRS and FHIR server to get
+# ready.
+
 set -e
 
+#######################################
+# Function that defines the endpoints
+# Globals:
+#   OPENMRS_URL
+#   SINK_SERVER
+# Arguments:
+#   Flag whether to use docker
```

Comment:
ditto about the comment line length; here and elsewhere.

Reply:
Fixed